docs/data/charts/styling/styling.md

```diff
@@ -42,6 +42,71 @@ Here is an example of the d3 Categorical color palette.
 
 {{"demo": "ColorTemplate.js"}}
 
+### Values color
+
+Colors can also be set according to item values.
+To do so, use the `colorMap` property of the corresponding axis.
+
+Learn more on how to use this feature with each chart component on their dedicated docs section:
+
+- [bar chart](/x/react-charts/bars/#color-scale)
```

Comment:
You sometime use plural, sometime singular for for these (bar chart vs bar charts, same for line and scatter).
It would be nice to unify

Reply:
Intuitively, I tend to use:

- plural for charts in general (The bar charts have options for ...)
- singular if describing the impact of a feature because I consider the chart the dev is working on "when clicking on the chart, ..."

```md
Bar charts provides two click handlers:

- `onItemClick` for click on a specific bar.
- `onAxisClick` for a click anywhere in the chart
```

But could be simplified with only plural/singular @samuelsycamore do you have some insight on this point?

Reply:
If there is an internal coherence I'm good, but yeah @samuelsycamore opinion will have more weight :ok_hand: